### PR TITLE
Update to 1.19

### PR DESCRIPTION
**Pull request by JRoy**
- [x] Mobs
- [x] API/VersionUtil/README
- [x] items.json
  - https://github.com/EssentialsX/ItemDbGenerator/tree/mc/1.19 	

for the new chat stuff, I'll tackle that once we do adventure support (#4717) in 2.20 or I may use something md_5 hacks into his api if its easy enough

thanks for your concern @broccolai

### CODE REVIEW

**Comment by JRoy** [comment]
WHOEVER MERGES THIS PLEASE USE REBASE AND MERGE

**Review by sando1234 — CHANGES_REQUESTED** [review]
I found nothing wrong :D.

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ however, have some new requirements:
 * **EssentialsX requires CraftBukkit, Spigot or Paper to run.** Other server software may work, but these are not tested
   by the team and we may not be able to help with any issues that occur.
 * **EssentialsX currently supports Minecraft versions 1.8.8, 1.9.4, 1.10.2, 1.11.2, 1.12.2, 1.13.2, 1.14.4, 1.15.2, 
-  1.16.5, 1.17.1, and 1.18.2.**
+  1.16.5, 1.17.1, 1.18.2, 1.19.**
```
Comment (mdcfe) [inline README.md]: and

**Review by mdcfe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (codebycam) [reply README.md]: LGTM

**Review by mdcfe — APPROVED** [review]

**Pull request merged**